server/server/src/billing/stripe_client.rs

```diff
@@ -0,0 +1,273 @@
+use crate::billing::stripe_client::SimplifiedStripeError::{
+    CardDeclined, InvalidCreditCard, Other,
+};
```

Comment:
If you're going to do this, just put `use SimplifiedStripeError::*` underneath the enum itself.